purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -146,6 +146,9 @@ internal class BillingWrapper(
             } else {
                 executePendingRequests()
             }
+        } else {
+            // This shouldn't happen, but if it does, we want to propagate an error instead of hanging.
+            request(PurchasesError(PurchasesErrorCode.UnknownError, "BillingWrapper is not attached to a listener"))
```

Comment:
Added this so it's more apparent if this ever happens in the future, but after the other fix, it shouldn't happen.